editions/test/tiddlers/tests/data/functions/WikifiedFunctions.tid

```diff
@@ -0,0 +1,30 @@
+title: Functions/WikifiedFunctions
+description: Wikified functions
+type: text/vnd.tiddlywiki-multiple
+tags: [[$:/tags/wiki-test-spec]]
+
+title: Output
+
+\whitespace trim
+\function fn-buffalo(p)
+[<p>addsuffix[ with a buffalo]]
+\end
+
+\procedure proc-buffalo(p)
+<$text text={{{ [<p>addsuffix[ with a buffalo]] }}}/>
+\end
+
+\define macro-buffalo(p)
+$p$ with a buffalo
+\end
+
+<<fn-buffalo "Going to lunch">>
+
+<<proc-buffalo "Going to lunch">>
+
+<<macro-buffalo "Going to lunch">>
```

Comment:
hmm, I think, there should be different texts, so we can actually see a difference if something goes wrong

Reply:
> hmm, I think, there should be different texts, so we can actually see a difference if something goes wrong

On the contrary, I think it's actually easier to visually verify that the strings are all identical. There's no ambiguity because of the paragraph tags.